CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Added
 - [#1852](https://github.com/thanos-io/thanos/pull/1852) Add support for `AWS_CONTAINER_CREDENTIALS_FULL_URI` by upgrading to minio-go v6.0.44
 - [#1854](https://github.com/thanos-io/thanos/pull/1854) Update Rule UI to support alerts count displaying and filtering.
+- [#1904](https://github.com/thanos-io/thanos/pull/1904) Add a no-chunk option in Store Series API.
```

Comment:
Perhaps lets mention that this greatly improves performance of the `/api/v1/series` endpoint?

Reply:
Done